### ISSUE TITLE
percent-for-art_cpd

### ISSUE THREAD

**Issue by lsat12357** [OregonDigital/OD2-migration#351]
**Item Count:**  199 items

**Item Types:** Generic

**Access Restrictions:**

**Complex Objects:** 199

**pid List:** https://github.com/OregonDigital/OD2-migration/blob/master/percent-for-art/percent-for-art_cpd_pids
see #124

**Comment by lsat12357** [comment]
exported 20220807
migrated 20220808

**Comment by sseymore** [comment]
df67zm81n - This is fine. Convert to text single work.
df67zj58f -remediated.
df67zn867 - remediated.
df67zq355- replace cpd with this one and attach children https://oregondigital.org/catalog/oregondigital:fx71d861m
df67zq398 -  replace cpd with this one and attach children https://oregondigital.org/catalog/oregondigital:fx71d862w
df67zm76s -  I deleted this asset and the children. Converted to a pdf https://oregondigital.org/catalog/oregondigital:fx71d8635#page/1/mode/1up

**Comment by lsat12357** [comment]
I reexported the two remediated assets, exported the 3 new assets, and removed the 3 deprecated pids from the pidlist, which feels kind of wrong, but I am documenting it here.

**Comment by lsat12357** [comment]
ok i have a question about the two new cpds that were created (ie ^^)
 this one was missing type https://oregondigital.org/reviewer/oregondigital:fx71d861m
so I looked at another generic asset and added http://purl.org/dc/dcmitype/Collection
but then I looked at https://oregondigital.org/catalog/oregondigital:fx71d862w assuming it would have the same problem, but it has format pdf and type text.  Did you mean to assign those? Have I incorrectly set the type on fx71d861m?

**Comment by jsimic** [comment]
@lsat12357 No, you are right. [fx71d862w](https://oregondigital.org/catalog/oregondigital:fx71d862w) is wrong. Corrected.

**Comment by lsat12357** [comment]
verification ok (ignore page count df67zm81n, od1 has 2 ghost pages)
